src/components/panel/panel.js

```diff
@@ -103,13 +103,14 @@ angular
  *     values. The panel will not open until all of the promises resolve.
  *   - `attachTo` - `{Element=}`: The element to attach the panel to. Defaults
  *     to appending to the root element of the application.
+ *   - `autoWrap` - `{boolean=}`: Whether to automatically wrap the template
```

Comment:
Why make this an option? What if we just always add <md-panel>? And if we do, do we also need to wrap that in a containing div?